crates/viewer/re_selection_panel/src/selection_panel.rs

```diff
@@ -1322,4 +1327,156 @@ mod tests {
         harness.run();
         harness.snapshot("selection_panel_component_hybrid_overwrite");
     }
+
+    #[test]
+    fn selection_panel_view_snapshot() {
+        fn setup_blueprint(test_context: &mut TestContext) -> ViewId {
```

Comment:
find this extra fn a bit strange stylistically, but don't mind it 🤷 . Some deeper philosophy on structure at work?